ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,48 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+RSM:
```

Comment:
Why not use `SS` and mark the field `run_command` as missing? It seems this is not doing anything different than `SS`, its just that the field for the command doesnt exist or can't be passed to cloud.